docker-compose/elasticsearch/v8.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.8'
+
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.8.2
+    environment:
+      - discovery.type=single-node
+      - http.host=0.0.0.0
+      - transport.host=127.0.0.1
+      - xpack.security.enabled=false  # Disable security features
+      - xpack.security.http.ssl.enabled=false  # Disable HTTPS
```

Comment:
the settings look identical between v7 and v8, but in the script we were clearly customizing which options to pass

Reply:
ok i added some specific test for v7 and  v8